packages/unmock-core/src/service/interfaces.ts

```diff
@@ -100,6 +101,9 @@ export interface IObjectToService {
 }
 
 export interface IServiceCore {
+
+  transformer: (req: ISerializedRequest, o: OpenAPIObject) => OpenAPIObject;
```

Comment:
Is this rather a `transform` operation? Also wondering if this should be curried or not, without currying it's definitely easier to understand but just wondering if it would be more convenient.

Reply:
I had the exact same internal debate and arrived at the exact same question. I don't have an opinion...